src/borg/testsuite/archiver.py

```diff
@@ -1367,6 +1367,15 @@ def test_delete(self):
         with Repository(self.repository_path) as repository:
             self.assert_equal(len(repository), 1)
 
+    def test_delete_multiple(self):
+        self.create_regular_file('file1', size=1024 * 80)
+        self.cmd('init', '--encryption=repokey', self.repository_location)
+        self.cmd('create', self.repository_location + '::test1', 'input')
+        self.cmd('create', self.repository_location + '::test2', 'input')
+        self.cmd('create', self.repository_location + '::test3', 'input')
+        self.cmd('delete', self.repository_location + '::test1', 'test2')
+        self.cmd('extract', '--dry-run', self.repository_location + '::test3')
```

Comment:
interesting would also be to make sure `borg delete repo archive` does not delete the full repo.